### PR TITLE
fixed bug in score.R and added info to FAQ

### PR DESCRIPTION
**Pull request by LeandroRitter**

### CODE REVIEW

File: workflow/scripts/score.R [hunk 1]
```diff
@@ -55,7 +55,7 @@ if(as.numeric(df[1,2])>as.numeric(df[1,3]) & as.numeric(df[1,3])>as.numeric(df[1
 
 
 #PMD SCORES
-df<-try(read.delim(paste0(out_dir,"/",RefID,".PMDscores.txt"),header=FALSE,sep="\t"),silent=TRUE)
+df<-try(read.delim(paste0(output_dir,"/",RefID,".PMDscores.txt"),header=FALSE,sep="\t"),silent=TRUE)
```
Comment (percyfal) [inline workflow/scripts/score.R:58]: A general comment:  you could use the function `file.path` to generate path names. Then you wouldn't need to paste the separator "/"; in fact the separator would be inferred from the os. Also, from the docs: 

>      The implementation is designed to be fast (faster than paste) as
>     this function is used extensively in R itself.

File: workflow/scripts/score.R [hunk 2]
```diff
@@ -66,7 +66,7 @@ total_score<-total_score+1
 #READ LENGTH DISTRIBUTION
 if(file.exists(paste0(dir_with_name_list,"/",RefID,".read_length.txt")))
 {
-df<-as.numeric(readLines(paste0(out_dir,"/",RefID,".read_length.txt")))
+df<-as.numeric(readLines(paste0(output_dir,"/",RefID,".read_length.txt")))
```
Comment (percyfal) [inline workflow/scripts/score.R:69]: Ditto

**Review by percyfal — COMMENTED** [review]
@LeandroRitter I have a general comment regarding the use of `paste0` to generate path names; I suggest the use of `file.path` instead. cf comments to code.

**Comment by LeandroRitter** [comment]
@percyfal I found and fixed quite a few bugs in the score.R script. Looks like after another modification of the authentic.smk, the score.R was not modified accordingly so the scoring system was not working correctly for some period. I also tried to replace paste0 with file.path where possible without too much fiddling with the code as I am not entirely sure about the behavior of file.path in contrast to paste0. Hope it is ok for now, and I will try to switch more and more to file.path in the future

**Review by percyfal — APPROVED** [review]
LGTM!

**Pull request merged**